features/call/impl/src/main/kotlin/io/element/android/features/call/impl/ui/CallScreenView.kt

```diff
@@ -58,25 +61,36 @@ interface CallScreenNavigator {
 @Composable
 internal fun CallScreenView(
     state: CallScreenState,
+    pipState: PictureInPictureState,
     requestPermissions: (Array<String>, RequestPermissionCallback) -> Unit,
     modifier: Modifier = Modifier,
 ) {
+    fun handleBack() {
+        if (pipState.supportPip) {
+            pipState.eventSink.invoke(PictureInPictureEvents.EnterPictureInPicture)
+        } else {
+            state.eventSink(CallScreenEvents.Hangup)
+        }
+    }
+
     Scaffold(
         modifier = modifier,
         topBar = {
-            TopAppBar(
-                title = { Text(stringResource(R.string.element_call)) },
-                navigationIcon = {
-                    BackButton(
-                        imageVector = CompoundIcons.Close(),
-                        onClick = { state.eventSink(CallScreenEvents.Hangup) }
-                    )
-                }
-            )
+            if (!pipState.isInPictureInPicture) {
```

Comment:
Technically you could also pass the web view's coordinates to the PiP code instead so it only renders that part of the activity so the webview wouldn't need a resize. It's more work, so I'm not sure it's really worth it.